scalyr_agent/monitor_utils/k8s.py

```diff
@@ -111,9 +112,7 @@ def cache(global_config):
         @type config: A Scalyr Configuration object
     """
     # split comma delimited string of namespaces to ignore in to a list of strings
-    namespaces_to_ignore = []
-    for x in global_config.k8s_ignore_namespaces:
-        namespaces_to_ignore.append(x.strip())
+    # TODOcz:
```

Comment:
What is left todo here?

Reply:
Sorry, that was a left over marker for myself to modify some lines which I did.  That code was all dead there.

Reply:
Yeah I figured that was likely the case.